package.json

```diff
@@ -45,7 +45,7 @@
         "node-watch": "0.5.9",
         "request": "2.81.0",
         "nonce": "1.0.3",
-        "bignum": "0.12.5",
+        "bignum": "https://github.com/JaredTate/node-bignum.git",
```

Comment:
Same here, I think it would be nice to eventually get the dependencies under DigiByte-Core if we need them for DigiByte-Core provided code.